petastorm/reader.py

```diff
@@ -43,6 +43,11 @@
 # Make it easier to import as "from petastorm.reader import ReaderV2"
 ReaderV2 = ReaderV2
 
+# Ventilator will hold ventilating new rowgroup if the current amount of 'rowgroups in processing' is greater than
```

Comment:
Not sure I understand the explanation of why we use this number of items in the ventilation queue.